packages/paypal-fastlane-integration/src/PayPalFastlaneShippingAddressForm.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { memo, useEffect, useRef } from 'react';
+import {
+    Address,
+    CheckoutSelectors,
+    Country,
+    CustomerAddress,
+    FormField,
+    ShippingInitializeOptions,
+    ShippingRequestOptions,
+} from '@bigcommerce/checkout-sdk';
+
+import { TranslatedString, localizeAddress } from '@bigcommerce/checkout/locale';
+import {
+    Button,
+    ButtonSize,
+    ButtonVariant,
+    DynamicFormField,
+    Fieldset,
+    LoadingOverlay,
+} from '@bigcommerce/checkout/ui';
+
+import isPayPalCommerceFastlaneMethod from './is-paypal-commerce-fastlane-method';
+import PoweredByPayPalFastlaneLabel from './PoweredByPayPalFastlaneLabel';
+
+interface PayPalFastlaneStaticAddressProps {
+    address: Address;
+    formFields: FormField[];
+    isLoading: boolean;
+    methodId: string;
+    deinitialize(options?: ShippingRequestOptions): Promise<CheckoutSelectors>;
+    initialize(options?: ShippingInitializeOptions): Promise<CheckoutSelectors>;
+    onAddressSelect(address: Address): void;
+    onFieldChange(fieldName: string, value: string): void;
+    onUnhandledError?(error: Error): void;
+    countries?: Country[];
+}
+
+export interface PayPalFastlaneAddressComponentRef {
+    showAddressSelector?: () => Promise<CustomerAddress | undefined>;
+}
+
+const PayPalFastlaneShippingAddressForm = (props: PayPalFastlaneStaticAddressProps) => {
+    const {
+        address: addressWithoutLocalization,
+        methodId,
+        formFields,
+        isLoading,
+        initialize,
+        deinitialize,
+        onUnhandledError,
+        onFieldChange,
+        countries,
+    } = props;
+    const address = localizeAddress(addressWithoutLocalization, countries);
+
+
+    const paypalFastlaneShippingComponent = useRef<PayPalFastlaneAddressComponentRef>({});
+
+    const paypalCommerceFastlaneOptions = {
+        paypalcommercefastlane: {
+            onPayPalFastlaneAddressChange: (showPayPalFastlaneAddressSelector: any) => {
+                paypalFastlaneShippingComponent.current.showAddressSelector = showPayPalFastlaneAddressSelector;
+            },
+        },
+    };
+
+    const braintreeFastlaneOptions = {
+        // braintreefastlane: {
```

Comment:
Is this comment necessary?

Reply:
Yes, it will be uncommented as soon as sdk part will be ready.